Xamarin.Forms.Platform.Android/AppCompat/Platform.cs

```diff
@@ -184,9 +184,13 @@ public static SizeRequest GetNativeSize(VisualElement view, double widthConstrai
 		{
 			Performance.Start(out string reference);
 
-			// FIXME: potential crash
 			IVisualElementRenderer visualElementRenderer = Android.Platform.GetRenderer(view);
 
+			if (visualElementRenderer == null || visualElementRenderer.View.IsDisposed())
```

Comment:
Do we know when this happen?

Reply:
Not really, I see it in the stack traces of the mentioned issues and this code was implemented for the fast renderers but not here. So I guess it doesn't hurt :)